### ISSUE TITLE
the on-screen keyboard layout is broken when un-zooming

### ISSUE THREAD

**Issue by fabi1cazenave** [fabi1cazenave/webtypist#4]
The current CSS for the on-screen keyboard works OK for zoom levels ≥ 1, but it’s broken for zoom levels < 1.

**Comment by saqibtanveer-dev** [comment]
can you please mention the version of app. is it v2

**Comment by fabi1cazenave** [comment]
No, v2 works fine in that regards. And as the sole purpose of v1 was to support browsers lacking WebComponent support, we should drop it quickly.

Keeping this ticket open until v1 is removed from this repository.
